### PR TITLE
Mirror IE + Chrome -> Edge for api/ (N-S)

### PR DESCRIPTION
**Pull request by vinyldarkscratch**
This PR is a part of #5214, in mirroring Chromium data over to Edge as per the new Edge version.  This performs four tasks: setting the existing `true` values to `≤18`, mirroring the Chromium data to any entires marked as `null` whilst setting the version to `≤79`, mirroring the Chromium data to entries marked as `false`, and setting anything that's `true` in IE to `12` in Edge.  The majority of this data has been generated by the mirroring script, however due to the nature of the migration, there's also a good portion modified manually (I do plan on upgrading the mirroring script to work better for this).

### CODE REVIEW

File: api/NetworkInformation.json [hunk 1]
```diff
@@ -107,7 +107,8 @@
               "version_added": "38"
             },
             "edge": {
-              "version_added": false
+              "version_added": "79",
+              "notes": "Only supported in Edge OS"
```
Comment (Elchi3) [inline api/NetworkInformation.json]: "Edge OS" hehe :)
Please remove this note.

File: api/NetworkInformation.json [hunk 2]
```diff
@@ -397,7 +398,8 @@
...
```
Comment (Elchi3) [inline api/NetworkInformation.json]: Please remove this note.

File: api/OfflineAudioCompletionEvent.json [hunk 3]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/OfflineAudioCompletionEvent.json]: Please remove this note.

File: api/OfflineAudioContext.json [hunk 4]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/OfflineAudioContext.json]: Please remove this note.

File: api/OscillatorNode.json [hunk 5]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/OscillatorNode.json]: Please remove this note.

File: api/PannerNode.json [hunk 6]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/PannerNode.json]: Please remove this note.

File: api/PerformanceFrameTiming.json [hunk 7]
```diff
@@ -13,7 +13,8 @@
             "notes": "See <a href='https://crbug.com/120796'>Chrome bug 120796</a>"
           },
           "edge": {
-            "version_added": null
+            "version_added": false,
+            "notes": "See <a href='https://crbug.com/120796'>Edge bug 120796</a>"
```
Comment (Elchi3) [inline api/PerformanceFrameTiming.json]: Chromium bug seems more accurate.

File: api/PeriodicWave.json [hunk 8]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/PeriodicWave.json]: Please remove this note.

File: api/SpeechRecognitionErrorEvent.json [hunk 9]
```diff
@@ -1,173 +0,0 @@
-{
```
Comment (Elchi3) [inline api/SpeechRecognitionErrorEvent.json]: This file is removed? Why is that?

File: api/StereoPannerNode.json [hunk 10]
```diff
@@ -61,7 +61,8 @@
               "notes": "Before Chrome 59, the default values were not supported."
             },
             "edge": {
-              "version_added": null
+              "version_added": "≤79",
+              "notes": "Before Edge 59, the default values were not supported."
```
Comment (Elchi3) [inline api/StereoPannerNode.json]: Please remove this note.

File: api/SyncManager.json [hunk 11]
```diff
@@ -167,9 +167,16 @@
...
             ],
-            "edge": {
-              "version_added": false
-            },
+            "edge": [
+              {
+                "version_added": "79"
+              },
+              {
+                "version_added": "79",
```
Comment (Elchi3) [inline api/SyncManager.json]: Please remove this range.

File: api/NetworkInformation.json (hunk 1 above)
Comment (vinyldarkscratch) [reply api/NetworkInformation.json]: _Imagine if Microsoft DID make a web-only OS...  😛_ 

Should we also set the support itself to `false`?  If it's only supported in Chrome OS, seems like it wouldn't be supported in Edge at all.

Comment (Elchi3) [reply api/NetworkInformation.json]: :D 
Good point, yes, I agree to set it to `false`.

File: api/SpeechRecognitionErrorEvent.json (hunk 9 above)
Comment (vinyldarkscratch) [reply api/SpeechRecognitionErrorEvent.json]: Huh, usually my Git client doesn't have issues removing the correct files during merge conflicts...  I went ahead and added this file back!

**Pull request merged**